### ISSUE TITLE
ReadTimeout: HTTPSConnectionPool(host='api.telegram.org', port=443): Read timed out. (read timeout=30)

### ISSUE THREAD

**Issue by fuomag9** [fuomag9/ChaturbateBot#14]
https://sentry.io/share/issue/5abd2e5e94d44d8c912ffe5c3293f901/

```
timeout: The read operation timed out
(6 additional frame(s) were not displayed)
...
  File "http/client.py", line 258, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
  File "python3.6/socket.py", line 586, in readinto
    return self._sock.recv_into(b)
  File "python3.6/ssl.py", line 1009, in recv_into
    return self.read(nbytes, buffer)
  File "python3.6/ssl.py", line 871, in read
    return self._sslobj.read(len, buffer)
  File "python3.6/ssl.py", line 631, in read
    v = self._sslobj.read(len, buffer)

ReadTimeoutError: HTTPSConnectionPool(host='api.telegram.org', port=443): Read timed out. (read timeout=30)
(2 additional frame(s) were not displayed)
...
  File "urllib3/util/retry.py", line 357, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "urllib3/packages/six.py", line 686, in reraise
    raise value
  File "urllib3/connectionpool.py", line 601, in urlopen
    chunked=chunked)
  File "urllib3/connectionpool.py", line 389, in _make_request
    self._raise_timeout(err=e, url=url, timeout_value=read_timeout)
  File "urllib3/connectionpool.py", line 309, in _raise_timeout
    raise ReadTimeoutError(self, url, "Read timed out. (read timeout=%s)" % timeout_value)
https://sentry.io/share/issue/5abd2e5e94d44d8c912ffe5c3293f901/
ReadTimeout: HTTPSConnectionPool(host='api.telegram.org', port=443): Read timed out. (read timeout=30)
(9 additional frame(s) were not displayed)
...
  File "telebot/apihelper.py", line 54, in _make_request
    timeout=(connect_timeout, read_timeout), proxies=proxy)
  File "requests/sessions.py", line 508, in request
    resp = self.send(prep, **send_kwargs)
  File "raven/breadcrumbs.py", line 303, in send
    resp = real_send(self, request, *args, **kwargs)
  File "requests/sessions.py", line 618, in send
    r = adapter.send(request, **kwargs)
  File "requests/adapters.py", line 521, in send
    raise ReadTimeout(e, request=request)

ReadTimeout: HTTPSConnectionPool(host='api.telegram.org', port=443): Read timed out. (read timeout=30)
```

**Comment by liyiecho** [comment]
bot.polling(none_stop=True) should be in a loop. I used this to solve the problem. Here is the code:
```
 while True:
   try:
    bot.polling(none_stop=True)
   except Exception as e:
       handle_exception(e)
```

**Comment by fuomag9** [comment]
I implemented it in 7452a588d02e18e6fb5fc7925fb4966b95762784
